operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakIngressTest.java

```diff
@@ -57,22 +56,7 @@ public void testIngressOnHTTP() {
                 .withStrictBackchannel(false);
         if (isOpenShift) {
             kc.getSpec().setIngressSpec(new IngressSpecBuilder().withIngressClassName(KeycloakController.OPENSHIFT_DEFAULT).build());
-
-            // see https://github.com/keycloak/keycloak/issues/14400#issuecomment-1659900081
-            kc.getSpec().setUnsupported(new UnsupportedSpecBuilder()
-                    .withNewPodTemplate()
-                        .withNewSpec()
-                            .addNewContainer()
-                                .addNewEnv()
-                                    .withName("KC_PROXY")
-                                    .withValue("edge")
-                                .endEnv()
-                            .endContainer()
-                        .endSpec()
-                    .endPodTemplate()
-                    .build());
```

Comment:
We now use `proxy-headers=xforwarded` in all tests to reflect [default Nginx behaviour](https://kubernetes.github.io/ingress-nginx/user-guide/miscellaneous/). So this is not needed anymore.